src/main/java/javaClasses/homework_4/Lilit_Manukyan/Developer.java

```diff
@@ -0,0 +1,27 @@
+package javaClasses.homework_4.Lilit_Manukyan;
+
+public class Developer extends Staff {
+    private String techstack;
+    public Developer(String name, String lastName, String techstack) {
```

Comment:
Please just add one extra line to separate field from method. It will make better readability

Reply:
Done